en/docs/design/create-api/create-a-streaming-api-from-an-asyncapi-definition.md

```diff
@@ -0,0 +1,109 @@
+# Create a Streaming API from an AsyncAPI Definition
+
+An AsyncAPI definition is a format that describes asynchronous APIs. 
+
+The following types of Streaming APIs can be created using an existing AsyncAPI definition:
+
+1. [WebSocket]({{base_path}}/design/create-api/create-streaming-api/create-a-websocket-streaming-api)
+2. [WebSub (WebHooks)]({{base_path}}/design/create-api/create-streaming-api/create-a-websub-streaming-api)
+3. [Server Sent Events (SSE)]({{base_path}}/design/create-api/create-streaming-api/create-a-sse-streaming-api)
+
+Follow the instructions below to create a Streaming API in this manner.
+
+
+1. Sign in to the WSO2 API Publisher `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher`).
```

Comment:
```suggestion
1. Sign in to the WSO2 API Manager (WSO2 API-M) Publisher Portal `https://<hostname>:9443/publisher` (e.g., `https://localhost:9443/publisher`).
```

Reply:
Shall we change as above by introducing the abbreviation for API Manager? From here onwards you can directly use WSO2 API-M in the same page without referring to the full name.